src/Checkout/Step/CheckoutStepShippingMethod.php

```diff
@@ -55,11 +55,17 @@ public function ShippingMethodForm()
                 $order->setShippingMethod($estimates->First());
             }
 
+            // style label
+            $labels = [];
+            foreach ($estimates as $estimate) {
+                $labels[$estimate->ID] = $estimate->getTitle();
+            }
+            
             $fields->push(
                 OptionsetField::create(
                     "ShippingMethodID",
                     _t('CheckoutStep_ShippingMethod.ShippingOptions', 'Shipping Options'),
-                    $estimates->map(),
```

Comment:
What's the use of changing this? Does `getTitle` not get called? If so you can simply call in $estimates->map('ID', 'getTitle');

Reply:
Thanks for the tip.  Please see commit https://github.com/silvershop/silvershop-shipping/pull/69/commits/eca7def766f80195f711a9100af713f1ff2c301c that updates CheckoutStepShippingMethod.php.